src/software/jetson_nano/services/motor.cpp

```diff
@@ -124,13 +125,19 @@ MotorService::MotorService(const RobotConstants_t& robot_constants,
     reset_gpio.setValue(GpioState::HIGH);
     sleep(1);
 
-    // TMC6100 Setup
-    for (uint8_t i = 0; i < NUM_MOTORS; i++)
+    // Drive Motor Setup
+    for (uint8_t motor = 0; motor < NUM_DRIVE_MOTORS; motor++)
     {
-        startDriver(i);
-        checkDriverFault(i);
-        startController(i);
+        startDriver(motor);
+        checkDriverFault(motor);
```

Comment:
is it safe to ignore faults (besides logging them) ?

Reply:
if there is a fault set, then the motor itself won't spin and things will stop so its safe to continue

there will need to be some handling code however to detect if any of the circuits fault at any time to shutoff all motors, but that will come in the next pr

Reply:
Curious if when the motor stops working, will we update the robot's unavailable_capabilities_?

Reply:
yeah we can, there is no fault handling/saftety in this PR

I've got some recovery features coming soon :tm: and one of them will include detecting a stalled motor or a broken encoder to safety shutoff everything. maybe capabilities is a good solution, but i think even better would be if we didn't count that robot and treated it like we got a yellow card + an obstacle until the next stoppage